ui/src/sandbox.rs

```diff
@@ -651,10 +675,23 @@ pub mod fut {
             cmd
         }
 
-        fn miri_command(&self, req: impl EditionRequest) -> Command {
+        fn miri_command(
+            &self,
+            env_settings: Vec<(&'static str, String)>,
+            req: impl EditionRequest,
+        ) -> Command {
             let mut cmd = self.docker_command(None);
             cmd.apply_edition(req);
 
+            let miri_env =
+                if let Some((_, flags)) = env_settings.iter().find(|(k, _)| k == &"MIRIFLAGS") {
+                    format!("MIRIFLAGS={} -Zmiri-disable-isolation", flags)
```

Comment:
Changing the flags like this seems potentially confusing.

If people can set their own flag I am not sure if we even still need the default flag?

Reply:
That's a good point.

I'm concerned that if this flag is removed by default, that people will find programs that used to work and don't anymore. How would we direct them to add the magic comment? I can definitely add something to the help page, but is that enough?

Reply:
Stuff already breaks on the playground over time, as crate versions change. I especially hope there's nobody relying on miri-in-playground not changing in any real way...

Reply:
If you want to be really fancy, when Miri shows the error saying "pass -Zmiri-disable-isolation to run this operation", the playground could turn that into a link that adds the relevant magic command. ;) The playground has such links for other situations. Those are easier though since here, if there already is a MIRIFLAGS comment, one has to now *change* the existing comment...

Reply:
I could probably handle most cases. Would it be better to do this edit which is almost always correct, or append something to the output that points to documentation? Both options seem sort of bad in slight ways. I like that currently the playground _just_ echoes cargo output.